src/helpers/cellPath.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const BemCellStringify = require('@bem/sdk.naming.cell.stringify');
+
+/**
+ * Returns path to file of BEM-cell using naming rules
+ */
+module.exports = (bemCell, convention) => BemCellStringify(convention)(bemCell);
```

Comment:
точно не `naming.file.stringify`?

Reply:
I got `cell` (block/elem/.../tech/layer) and `level.path` (absolute) and want to build path to file. Now I do `level.path` + `..` (because layer part is already in cell) + `cell`

`naming.file.stringify` do `level` (any dir) + `cell`.

What is level ? Layer is not equal to level ? 

I don't think that:

```js
fileStringify(
    new BemFile({
        cell,
        level: level.path + '/..'
    })
)
```

is better enough.

Reply:
Started conversation in https://github.com/bem/bem-sdk/issues/278